### ISSUE TITLE
Blank Sprite Reset Option

### ISSUE THREAD

**Issue by elewental** [whiteblXK/ObjectBuilder#6]
from: https://github.com/ottools/ObjectBuilder/issues/37

Sometimes If I create a sprite that have a square fully transparent and drag there accidentally a wrong sprite, I need to remove it, then I have two options:
1 - create a 100% blank sprite to move there
2 - delete the whole object configuration and start again

That's why Im suggesting the creation of this button:

![image](https://user-images.githubusercontent.com/52132367/155870844-668b1687-b0d7-43b6-acf7-9d32cb3765dd.png)

or
![image](https://user-images.githubusercontent.com/52132367/155870851-b5c3895f-af93-4bef-a40a-71cc3a9220ca.png)

**Comment by ghost** [comment]
agreed, this is really frustrating when we make any mistake as you described, btw you don't need to create a new blank sprite for this, every object in tibia that use 100% blank sprite is using sprite number 0, the first sprite of the sheet

**Comment by elewental** [comment]
I didn't noticed that, thank you

**Comment by punkice3407** [comment]
Should I cherry pick commit from that issue? It will be ok?

**Comment by ghost** [comment]
> Should I cherry pick commit from that issue? It will be ok?

100% yes

**Comment by punkice3407** [comment]
added
